### ISSUE TITLE
Link error in Albany due to MueLu undefined reference on Ride/Waterman

### ISSUE THREAD

**Issue by ikalash** [trilinos/Trilinos#3772]
The past couple of days, the Albany builds on Ride/Waterman have been failing with a link error in MueLu:

http://cdash.sandia.gov/CDash-2-3-0/viewBuildError.php?buildid=77600
http://cdash.sandia.gov/CDash-2-3-0/viewBuildError.php?onlydeltap&buildid=77574

It looks like there is a parsing error:

```
undefined reference to Tpetra::CrsMatrix >::setAllValues(Kokkos::View, Kokkos::MemoryTraits<0u> > const&, Kokkos::View, Kokkos::MemoryTraits<0u> > const&, Kokkos::View, void> const&)'`
```

The angle brackets seem to be messed up.

@trilinos/muelu

**Comment by jhux2** [comment]
@ikalash Could you tell me the date of the last clean build?

**Comment by ikalash** [comment]
@jhux2 : unfortunately not since 10/19:

http://cdash.sandia.gov/CDash-2-3-0/index.php?project=Albany&date=2018-10-19

There were failures in the build after that every day due to either changes on the Albany or on the Trilinos side.  These were finally resolved last Fri. Oct. 26, and this error cropped up on Sat. Oct. 27: http://cdash.sandia.gov/CDash-2-3-0/viewBuildError.php?buildid=77530 .

**Comment by jewatkins** [comment]
The builds are working again. It looks like a similar issue was encountered in #3804 and was fixed by #3805. Thanks!
